src/app_ui/address.rs

```diff
@@ -34,20 +43,31 @@ pub fn ui_display_pk(addr: &[u8]) -> Result<bool, AppSW> {
     let addr_hex = from_utf8_mut(&mut addr_hex).unwrap();
     addr_hex[2..].make_ascii_uppercase();
 
-    let my_field = [Field {
-        name: "Address",
-        value: addr_hex,
-    }];
-
-    let my_review = MultiFieldReview::new(
-        &my_field,
-        &["Confirm Address"],
-        Some(&EYE),
-        "Approve",
-        Some(&VALIDATE_14),
-        "Reject",
-        Some(&CROSSMARK),
-    );
-
-    Ok(my_review.show())
+    #[cfg(not(any(target_os = "stax", target_os = "flex")))]
+    {
+        let my_field = [Field {
+            name: "Address",
+            value: addr_hex,
+        }];
+
+        let my_review = MultiFieldReview::new(
+            &my_field,
+            &["Confirm Address"],
+            Some(&EYE),
+            "Approve",
+            Some(&VALIDATE_14),
+            "Reject",
+            Some(&CROSSMARK),
+        );
+
+        Ok(my_review.show())
+    }
```

Comment:
use same wording as for C boilerplate

- "Verify _appname_ address" instead of "Confirm address"
- "Cancel" instead of "reject"
- "Confirm" instead of "approve"